lib/components/notifications-component.js

```diff
@@ -15,9 +16,23 @@ export default class NotificationsComponent extends BaseContainer {
 		let text = this.driver.findElement( by.css( '.wpnc__notes' ) ).getText();
 		return text;
 	}
+
 	selectCommentByText( commentText ) {
-		const commentSelector = by.xpath( `//div[normalize-space(text())='${commentText}']` );
-		return driverHelper.clickWhenClickable( this.driver, commentSelector );
+		const self = this;
+		const commentSelector = by.css( 'div > div.wpnc__text-summary' );
+		let comments = this.driver.findElements( by.css( 'div.wpnc__excerpt' ) );
+		this.driver.wait( until.elementLocated( commentSelector ), self.explicitWaitMS, 'Could not locate comment exerpt.' );
+		webdriver.promise.filter( comments, function( comment ) {
+			return comment.getText().then( function( text ) {
+				return text === commentText;
+			} );
+		} ).then( function( filteredComments ) {
+			if ( filteredComments.length === 1 ) {
+				return driverHelper.clickWhenClickable( self.driver, commentSelector );
```

Comment:
Thank you for the suggestions! After our chat early today, I ended up deciding to revert back to the original xpath selector as it was more accurate for selecting the comment text in this case.